parcel.go

```diff
@@ -14,47 +15,135 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("insert into parcel (client, status, address, created_at) values (:Client, :Status, :Address, :CreatedAt)",
+		sql.Named("Client", p.Client),
+		sql.Named("Status", p.Status),
+		sql.Named("Address", p.Address),
+		sql.Named("CreatedAt", p.CreatedAt))
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	if err != nil {
+		return 0, err
+	}
+	lastID, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+
+	// Возвращаем идентификатор
+	return int(lastID), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
+	// Выполняем запрос на получение данных по указанному number
+	row := s.db.QueryRow("SELECT number, client, status, address, created_at FROM parcel WHERE number = ?", number)
 
-	// заполните объект Parcel данными из таблицы
+	// Заполняем объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+
+	// Если возникла ошибка (например, записи нет), возвращаем её
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```